draft-ietf-rats-corim.md

```diff
@@ -1171,7 +1177,9 @@ If the search criteria are satisfied, the `endorsements` entries are asserted wi
 
 #### Conditional Endorsement Series Triple {#sec-comid-triple-cond-series}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/310
+A Conditional Endorsement Series triple uses a stateful environment, (i.e. stateful-environment-record), that identifies a Target Environment based on an environment-map plus the measurement-map measurements that have matching Evidence.	
+
+The series object is an array of conditional-series-record that has both Reference and Endorsed Values. Each conditional-series-record record is evaluated in the order it appears in the series array. The Endorsed Values are accepted if the series condition in a conditional-series-record matches the ACS.  The first conditional-series-record that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted. If none of the series conditions match an ACS Entry, the triple is not matched, and no Endorsed values are accepted.	
```

Comment:
```suggestion
The series object is an array of `conditional-series-record` that has both Reference and Endorsed Values.
Each `conditional-series-record` record is evaluated in the order it appears in the series array.
The Endorsed Values are accepted if the series condition in a `conditional-series-record` matches the ACS.
The first `conditional-series-record` that successfully matches an ACS Entry terminates the matching and the corresponding Endorsed Values are accepted.
If none of the series conditions match an ACS Entry, the triple is not matched, and no Endorsed values are accepted.	
```

We still have the problem of needing to rule out negative matches based on lack of information. Please leave a link to https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/321

Reply:
Done, I want to progress this PR first so that we  can discuss the Issue in #321 
@deeglaze  please check